src/status_im2/navigation/core.cljs

```diff
@@ -264,8 +264,8 @@
                  (fn [] (dissmiss-overlay "visibility-status-popover")))
 
 ;; BOTTOM SHEETS
-(re-frame/reg-fx :show-bottom-sheet (fn [] (show-overlay "bottom-sheet")))
-(re-frame/reg-fx :hide-bottom-sheet (fn [] (dissmiss-overlay "bottom-sheet")))
+(re-frame/reg-fx :display-bottom-sheet (fn [] (show-overlay "bottom-sheet")))
```

Comment:
i meant it should be similar names hide-bottom-sheet-overlay and show-bottom-sheet-overlay